src/app/components/client/exposure_card/ExposureCard.stories.tsx

```diff
@@ -3,22 +3,67 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import type { Meta, StoryObj } from "@storybook/react";
-import { ExposureCard } from "./ExposureCard";
+import { Exposure, ExposureCard } from "./ExposureCard";
 import FamilyTreeImage from "../assets/familytree.png";
 import TwitterImage from "../assets/twitter-icon.png";
 import {
   createRandomBreach,
   createRandomScanResult,
 } from "../../../../apiMocks/mockData";
+import { StaticImageData } from "next/image";
+import { ReactNode } from "react";
+import { FeatureFlagName } from "../../../../db/tables/featureFlags";
+import {
+  defaultExperimentData,
+  ExperimentData,
+} from "../../../../telemetry/generated/nimbus/experiments";
+
+type ExposureCardWrapperProps = {
+  exposureData: Exposure;
+  exposureImg?: StaticImageData;
+  locale?: string;
+  isPremiumUser?: boolean;
+  isEligibleForPremium?: boolean;
+  resolutionCta?: ReactNode;
+  isExpanded?: boolean;
+  enabledFeatureFlags?: FeatureFlagName[];
+  experimentData?: ExperimentData;
+  removalTimeEstimate?: number;
+  onToggleExpanded?: () => void;
+};
```

Comment:
Is this not just `Partial<ExposureCardProps>`? Is that even needed?

Reply:
Removed with https://github.com/mozilla/blurts-server/pull/5189#discussion_r1806762503.